pkg/plugins/discovery_unix.go

```diff
@@ -2,5 +2,25 @@
 // +build !windows
 
 package plugins // import "github.com/docker/docker/pkg/plugins"
+import (
+	"github.com/docker/docker/pkg/homedir"
+	"github.com/docker/docker/rootless"
+	"path/filepath"
+)
 
 var specsPaths = []string{"/etc/docker/plugins", "/usr/lib/docker/plugins"}
+
+// SpecsPaths returns
+// { "%programdata%\docker\plugins" } on Windows,
+// { "/etc/docker/plugins", "/usr/lib/docker/plugins" } on Unix in non-rootless mode,
+// { "$XDG_DATA_HOME/.local/share/docker/plugins" } on Unix in rootless mode.
```

Comment:
Shouldn't this be {`$XDG_CONFIG_HOME/docker/plugins`, `$HOME/.local/lib/docker/plugins`} ?

Reply:
Yes, I changed the returned paths to your suggestion.
I additionally made the fallback (if the corresponding environment variable is not set) more explicit in the comment.